plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts

```diff
@@ -430,22 +439,23 @@ describe('ingester', () => {
         })
 
         it('should not be affected by other partitions ', async () => {
+            console.log('test start')
             await ingester.handleEachBatch([
-                createMessage('sid1', 1), // offset 1
-                createMessage('sid2', 2), // offset 2
-                createMessage('sid2', 2), // offset 3
+                createMessage('sid1', 1),
+                createMessage('sid2', 2),
+                createMessage('sid2', 2),
             ])
 
             await ingester.sessions[`${team.id}-sid1`].flush('buffer_age')
-            await ingester.handleEachBatch([createMessage('sid1', 1)]) // offset 4
+            await ingester.handleEachBatch([createMessage('sid1', 1)])
 
-            // We should now have a blocking session on partition 1 and 2 with partition 1 being commitable
+            // We should now have a blocking session on partition 1 and 2 with partition 1 being committable
             await commitAllOffsets()
             expect(mockCommit).toHaveBeenCalledTimes(1)
             expect(mockCommit).toHaveBeenLastCalledWith(
                 expect.objectContaining({
                     partition: 1,
-                    offset: 4,
+                    offset: 2,
```

Comment:
so, were the offsets before incorrect or the test setup starts at a different offset?

Reply:
The tests were just too simple. We incremented a global offset. Now we have a simulate per-partition offset tracker.

The tests themselves were fine, just needed changing to match this new format.